src/Middleware/src/Headstart.API/Controllers/DevOpsController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using OrderCloud.Catalyst;
+using OrderCloud.Integrations.EnvironmentSeed.Commands;
+using OrderCloud.Integrations.EnvironmentSeed.Models;
+
+namespace Headstart.API.Controllers
+{
+    [Route("devops")]
+    public class DevOpsController : CatalystController
+    {
+        private readonly IUploadTranslationsCommand command;
+
+        public DevOpsController(IUploadTranslationsCommand command)
+        {
+            this.command = command;
+        }
+
+        /// <summary>
+        /// Call this endpoint to upload translation files from source.
+        /// </summary>
+        [HttpPut, Route("translations")]
+        public async Task UploadTranslations([FromForm] UploadTranslationsRequest translationsRequest)
+        {
+            await command.UploadTranslationsFile(translationsRequest);
+        }
```

Comment:
I think this method is good and will be a lot more useful if we were to associate some kind of admin UI with the functionality here but we probably also want another command and endpoint that updates all translation files based on what is in the current source code. Two reasons for this

1. Anyone that is following along with headstart updates can call this endpoint to quickly retrieve updated translations
2. There is a use case I see quite commonly where translations are tied to a release and updating the translations  is generally a developer task. Such a flow could be created by calling this endpoint as part of the final step in a release pipeline.

Reply:
Good points here.

1. Adding a 'global' configuration section in the admin app will allow admin business users to perform actions that currently require developer intervention.
2. I briefly considered this, but didn't review the release pipeline to understand actions currently being performed. In any case, this is easy additional endpoint is easy to add.

Reply:
The release pipeline will need to be manually updated to resolve [Update translation files on deploy based on the saved translation file](#294). Additional details in linked issue.